pkg/resource/events-logger-config/alloy/events-logger-config.alloy.yaml.template

```diff
@@ -0,0 +1,28 @@
+# This file was generated by logging-operator.
+# It configures Alloy to be used as events logger.
+# - configMap is generated from events-logger.alloy.template and passed as a string
+#   here and will be created by Alloy's chart.
+# - Alloy runs as a deployment, with only 1 replica.
+alloy:
+  alloy:
+    configMap:
+      create: true
+      content: |-
+        {{ .AlloyConfig | nindent 8 }}
+    envFrom:
+    - secretRef:
+        name: {{ .SecretName }}
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: false
+      runAsUser: 10
+      runAsGroup: 10
+      runAsNonRoot: true
+      seccompProfile:
+        type: RuntimeDefault
+  controller:
+    type: deployment
+    replicas: 1
```

Comment:
You need to set crds.create: false as well right?

Reply:
I don't think we need. I checked the values used by `alloy-logs` and the default ones in our `alloy` repo and we never specify the crd field. Moreover I don't see any alloy CRD in the clusters where `alloy-logs` and `alloy-metrics` are enabled.

Reply:
Oh ok my bad.